src/mokka/src/main/java/pl/hycom/mokka/stubbing/WireMockStubMappingConverter.java

```diff
@@ -46,6 +47,11 @@ public StubMapping convert(MockConfiguration mockConfiguration) {
             responseDefinitionBuilder.withFixedDelay(mockConfiguration.getTimeout());
             responseDefinitionBuilder.proxiedFrom(mockConfiguration.getProxyBaseUrl());
 
+            if(mockConfiguration.getConfigurationContent() instanceof GroovyConfigurationContent) {
+                responseDefinitionBuilder.withTransformers("groovy-transformer");
```

Comment:
Maybe it would be good to define public static final String property on `GroovyResponseTransformer` and just use it everywhere?